### DISCUSSION TITLE
Add overflow toolbar like functionality

### DISCUSSION THREAD

**Discussion by jpzwarte** [storybookjs/storybook#29699] [Ideas]
### Is your feature request related to a problem? Please describe.

![CleanShot 2024-11-25 at 09 51 51@2x](https://github.com/user-attachments/assets/54586cfd-49f8-432b-b067-2fc986e7f509)

### Describe the solution you'd like

![CleanShot 2024-11-25 at 09 53 05@2x](https://github.com/user-attachments/assets/99a66bf7-bacc-4e4c-a0af-93ac4c498f1a)

When items do not fit anymore in the available space, overflow them in a menu, as seen in the screenshots.

### Are you able to assist to bring the feature to reality?

no

**Comment by domyen** [comment]
Thanks for the suggestion! We could probably extend the existing addon panel pattern to the toolbar as well. 

Would a community member be willing to PR this?

![2024-12-02 10 35 55](https://github.com/user-attachments/assets/0e7af242-e335-4bb8-b8b0-da172e2ebfcd)

**Reply by jpzwarte** [reply]
I would like to, but i'm not a React developer and i don't know how much of this task is pure React?
